airbyte-integrations/csv-destination/src/main/resources/spec.json

```diff
@@ -0,0 +1,20 @@
+{
+  "destinationId": "",
+  "destinationSpecificationId": "",
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/local-csv",
+  "specification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "CSV Destination Spec",
+    "type": "object",
+    "required": ["destination_path"],
+    "additionalProperties": false,
+    "properties": {
+      "destination_path": {
+        "description": "Path to the directory where csv files will be written. Must start with the local mount \"/local\". Any other directory appended on the end will be placed inside that local mount.",
```

Comment:
It'd be great to link to documentation that shows how/where this is configured and what the defaults are.